crypto/hash_test.go

```diff
@@ -7,6 +7,18 @@ import (
 )
 
```

Comment:
shouldn't we keep a test for the old deprecated version as well as long as it's supposed to work?

Reply:
Good point, thanks !